homeassistant/components/roomba/sensor.py

```diff
@@ -114,6 +119,20 @@ class RoombaSensorEntityDescription(
         value_fn=lambda self: self.run_stats.get("nScrubs"),
         entity_registry_enabled_default=False,
     ),
+    RoombaSensorEntityDescription(
+        key="total_cleaned_area",
+        translation_key="total_cleaned_area",
+        icon="mdi:texture-box",
+        native_unit_of_measurement=AREA_SQUARE_METERS,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda self: (
+            self.run_stats.get("sqft") * 9.29
+            if self.run_stats.get("sqft") is not None
```

Comment:
```suggestion
            if self.run_stats.get("sqft")
```
last thing, than this good to go if the docs PR is there

Reply:
Well, maybe that was a mistake on my part, sqft could be zero, without `is not None` this would return `None` and make the sensor `unknown`

Reply:
Could you please revert this change, sorry about that, my fault

Reply:
Yes will revert it. No problem.